pkg/admin/server/apis.go

```diff
@@ -136,6 +136,28 @@ func configDump(w http.ResponseWriter, r *http.Request) {
 
 }
 
+func envoyConfigDump(w http.ResponseWriter, r *http.Request) {
```

Comment:
It is for `/config_dump` like envoy, instead of mosn `/api/v1/config_dump`, in order to be compatible with `istioctl proxy-status xxx`.
Refer:
https://istio.io/latest/docs/ops/diagnostic-tools/proxy-cmd/